mappings/net/minecraft/network/packet/c2s/play/QueryChatPreviewC2SPacket.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_7496 net/minecraft/network/packet/c2s/play/QueryChatPreviewC2SPacket
```

Comment:
Could this class be named `RequestChatPreviewC2SPacket` to match the existing `RequestCommandCompletionsC2SPacket` class?

Reply:
It has a `queryId` record field, but I suppose that also works.